harness/determined/cli/master.py

```diff
@@ -10,7 +10,7 @@
 
 @authentication.required
 def config(args: Namespace) -> None:
-    response = api.get(args.master, "config")
+    response = api.get(args.master, "api/v1/master/config")
```

Comment:
nit we could be using the bindings here now but feel free to leave it for later.

Reply:
I'll create a small ticket to come back and use bindings here

Reply:
It was actually a super quick swap, already taken care of